src/map.h

```diff
@@ -838,6 +838,20 @@ class map
                           bool destroy = false, bool bash_floor = false,
                           const vehicle *bashing_vehicle = nullptr );
 
+        /**
+         * Knock down a tree (if present), creating stump and trunks.
```

Comment:
The function doesn't check for presence of a tree.

Reply:
Well, that's certainly suboptimal. Update incoming.